apps/postgres.go

```diff
@@ -0,0 +1,295 @@
+package apps
+
+import (
+	"fmt"
+	"github.com/openebs/openebs-e2e/common"
+	"github.com/openebs/openebs-e2e/common/e2e_config"
+	"github.com/openebs/openebs-e2e/common/k8stest"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+/*
+Before you start, ensure that you have at least one node without huge pages and Mayastor installed. You can use the
```

Comment:
> one node without huge pages and Mayastor installed

What's the reason to not use in mayastor node here? Is it a kernel crash?
I had to run postgres on a mayastor node for this ticket: https://github.com/openebs/mayastor/issues/1693
and I was able to run it with this config:
```
global:
  storageClass: mayastor-nvmf-3
image:
  tag: 15-debian-12
  debug: true
tls:
  enabled: true
  autoGenerated: true
primary:
  extendedConfiguration: |-
    huge_pages = off
  extraVolumeMounts:
    - name: pg-sample-config
      mountPath: /opt/bitnami/postgresql/share/postgresql.conf.sample
      subPath: postgresql.conf.sample
  extraVolumes:
    - configMap:
        name: pg-sample-config
      name:  pg-sample-config
  resourcesPreset: "medium"
  networkPolicy:
    enabled: false
  service:
    type: LoadBalancer
    externalTrafficPolicy: Local
  persistence:
    size: 9Gi
extraDeploy:
  - apiVersion: v1
    kind: ConfigMap
    metadata:
      name: pg-sample-config
    data:
      postgresql.conf.sample: |-
        huge_pages = off
volumePermissions:
  enabled: true
```

Reply:
We are using helm and there is no possibility to add huge_pages = off we tried it with the `extendedConfiguration`, but without success

Reply:
This was with helm as well; probably you were missing `extraDeploy` ?

Reply:
Let's leave it as it is for now, and then we can explore and update the whole process, as we've spent a huge amount of time with all the repository changes, updates for other engines, etc.